CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ Release date: TBD
 ### Bug Fixes
  - Fixed wrong cursor for "Edit" and "Remove" in Setting page
  - Fixed an issue where "Zoom in/out" does not properly work
+ - The "Allow mixed content" setting used to disable multiple security features to allow http content
```

Comment:
Cool, wondering how we might be able to translate it to a non-technical end user or admin. Would this be accurate?

```
The "Allow mixed content" setting that enables http content no longer disables existing features such as YouTube previews
```